### PR TITLE
Use -Werror when building in Travis CI

### PR DESCRIPTION
**Pull request by hughsie**
Do NOT do this for normal package builds.

### CODE REVIEW

File: contrib/debian/rules [hunk 1]
```diff
@@ -26,7 +26,7 @@ override_dh_auto_configure:
 	else \
 		export DELL="-Denable-dell=false -Denable-synaptics=false"; \
 	fi; \
-	dh_auto_configure -- $$UEFI $$DELL -Denable-dummy=true --libexecdir=/usr/lib
+	dh_auto_configure -- $$UEFI $$DELL -Denable-dummy=true -Denable-werror=true --libexecdir=/usr/lib
```
Comment (superm1) [inline contrib/debian/rules]: This is going to actually affect the distro build (it uses the same debian/rules).  This needs to get pulled up a little higher to `./contrib/ci/build_and_install_debs.sh`.  Maybe via an environment variable and `contrib/debian/rules` picks up that environment variable.

Comment (superm1) [reply contrib/debian/rules]: Actually the CI variable is already set.  I'll adjust this and I think I can fix it to bubble around.

Comment (hughsie) [reply contrib/debian/rules]: Please feel free to push to my wip branch, thanks.

**Comment by superm1** [comment]
This looks good to me now.

**Pull request merged**